.kitchen.yml

```diff
@@ -33,22 +31,54 @@ suites:
     run_list:
     - recipe[minitest-handler]
     - recipe[duplicity_ng::package]
-
+    - recipe[duplicity_ng::install_azure]
+    - recipe[duplicity_ng::install_boto]
+    - recipe[duplicity_ng::install_ftp]
+    - recipe[duplicity_ng::install_paramiko]
+    - recipe[duplicity_ng::install_par2]
+    - recipe[duplicity_ng::install_swift]
+    attributes:
+      duplicity_ng:
+        use_pip: false
+  - name: install_package_pip
+    run_list:
+    - recipe[minitest-handler]
+    - recipe[duplicity_ng::package]
+    - recipe[duplicity_ng::install_azure]
+    - recipe[duplicity_ng::install_boto]
+    - recipe[duplicity_ng::install_ftp]
+    - recipe[duplicity_ng::install_paramiko]
+    - recipe[duplicity_ng::install_par2]
+    - recipe[duplicity_ng::install_swift]
+    attributes:
+      duplicity_ng:
+        use_pip: true
   - name: install_source
     run_list:
     - recipe[minitest-handler]
     - recipe[duplicity_ng::source]
+    - recipe[duplicity_ng::install_azure]
+    - recipe[duplicity_ng::install_boto]
+    - recipe[duplicity_ng::install_ftp]
+    - recipe[duplicity_ng::install_paramiko]
+    - recipe[duplicity_ng::install_par2]
+    - recipe[duplicity_ng::install_swift]
     attributes:
       duplicity_ng:
-        path: /usr/bin/duplicity
```

Comment:
Yes, cross platform issues I mentioned in 72090c9 "... Fix duplicity paths under source suites, can either be /usr/bin/local/duplicity under debian family or /usr/bin/duplicity under rhel-ish." 

Either way, regardless of install method or platform, you would expect it to end up in the path?